bootstrap/src/test/java/io/airlift/bootstrap/DestroyExceptionInstance.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2010 Proofpoint, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.airlift.bootstrap;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.inject.Inject;
+
+public class DestroyExceptionInstance
+{
+    @Inject
+    public DestroyExceptionInstance(DependentInstance ignored)
+    {
+    }
+
+    @PostConstruct
+    public void postDestroyException()
+    {
+        TestLifeCycleManager.note("postDestroyException");
```

Comment:
- `postDestroyException` -> `post*Construct*...` ?
- did you intent to have `throw ` here? Is this method needed at all?

Reply:
Not strictly, was just following the convention of other instances. I'll remove.